R/servers.R

```diff
@@ -63,7 +63,8 @@ servers <- function(local = FALSE) {
   if (local) {
     locals
   } else {
-    rbind(locals, as.data.frame(shinyappsServerInfo(), stringsAsFactors = FALSE))
+    locals <- rbind(locals, as.data.frame(shinyappsServerInfo(), stringsAsFactors = FALSE))
```

Comment:
Please update the docs for `servers`, which indicates that there is always a single shinyapps.io entry; we should do something similar for cloud.